app/assets/stylesheets/components/_background.scss

```diff
@@ -1,6 +1,7 @@
 .bg-gray-lighter { background-color: $gray-lighter; }
 .bg-light-blue { background-color: $blue-light; }
 .bg-lightest-blue { background-color: $blue-lightest; }
+.bg-lightest-red { background-color: #fff7f8; }
```

Comment:
Since we're using `#fff7f8` in multiple places, should we add it to `app/assets/stylesheets/variables/_colors.scss` (`$red-lightest`)?